docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md

```diff
@@ -0,0 +1,208 @@
+---
+type: docs
+title: "Platform9 Managed Kubernetes (PMK) cluster"
+linkTitle: "Platform9 Managed Kubernetes (PMK) cluster"
+weight: 1
+description: >
+---
+
+## Deploy a Platform9 Managed Kubernetes cluster and connect it to Azure Arc
+
+The following document will guide on how to deploy a Kubernetes cluster with [Platform9 Managed Kubernetes (PMK)](https://platform9.com/managed-kubernetes/) and have it as a connected Azure Arc Kubernetes resource.
+With PMK, you can have your clusters deployed on-premises, in public clouds or at the edge. In this document, we'll explain the steps on how to create an **on-premise [BareOS](https://platform9.com/docs/kubernetes/bareos-what-is-bareos) cluster** using **PMK** and connect it to **Microsoft Azure Arc**.
+
+## Prerequisites
+
+* An Ubuntu 20.04 installed physical machine or VM. For this demonstration, we would be using single node kubernetes cluster installation. This machine would also act as our management host.
+
+* The [kubectl](https://platform9.com/learn/tutorials/kubectl) exe for cluster management locally. If you would want to have a seperate host for cluster management, follow the below steps to [install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management) on it.
+
+  ```shell
+  sudo apt-get update
+  sudo apt-get install -y apt-transport-https ca-certificates curl
+  sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+  echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+  sudo apt-get update
+  sudo apt-get install -y kubectl
+  ```
+
+  > **Note:** The *kubectl* package is installed from Kubernetes repository, hence the Google Cloud public signing key need to be downloaded to enable the repository.
+
+  > **All PMK cluster nodes would have kubectl installed once cluster is deployed.**
+
+* [Azure CLI (az)](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) version 2.15.0 and above. This cli tool would help to setup the Azure resources and connect the Kubernetes cluster to Azure Arc.
```

Comment:
This is not a recommendation, it's mandatory. Please [replicate](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/aks/aks_arm_template/#prerequisites)

![image](https://user-images.githubusercontent.com/46730876/139537845-24d95906-1aa7-40c0-83c5-ba430ea0e589.png)